.github/workflows/scripts/issues/statusLabelHandler.js

```diff
@@ -12,14 +12,23 @@ module.exports = async ({ core, context, github }) => {
     const repo = context.repo.repo;
     const issueNumber = context.issue.number;
 
+    core.debug(`>>> Context: ${JSON.stringify(context)}`);
+
     const issue = await github.rest.issues.get({
       owner,
       repo,
       issue_number: issueNumber,
     });
 
+    core.debug(`>>> Issue data: ${JSON.stringify(issue.data)}`);
+
     const issueAuthor = issue.data.user.login;
```

Comment:
I get that user can be undefined locally

```suggestion
    const issueAuthor = issue.data.user?.login;
```